src/control/run_go_tests.sh

```diff
@@ -56,22 +56,22 @@ if [ "$check" == "false" ]; then
 	setup_environment
 fi
 
-DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
+# allow extra args to be set manually
+TESTARGS=${TESTARGS:-""}
+DIR=$(readlink -f $(dirname ${BASH_SOURCE[0]}))
```

Comment:
(lint) Quote this to prevent word splitting. [SC2046]
(lint) Double quote to prevent globbing and word splitting. [SC2086]